server/models/paymentCard.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const paymentCardSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true,
+    },
+    lastName: {
+        type: String,
+        required: true,
+    },
+    cardNumber: {
+        type: Number,
+        required: true,
+        min: 8,
+        max: 16,
+    },
+    expDate: {
+        month: {type: String, required: true, min: 2, max: 2},
+        year: {type: String, required: true, min: 2, max: 2},
+    },
+})
```

Comment:
We aren't going to be storing credit card information in our database but rather we'll using Stripe and creating our customer data there. We can remove this whole model! Good job on thinking in the future on this one though! 👌🏻